bitnami/moodle/4/debian-11/rootfs/opt/bitnami/scripts/libmoodle.sh

```diff
@@ -388,14 +388,20 @@ moodle_configure_wwwroot() {
     # sed replacement notes:
     # - The ampersand ('&') is escaped due to sed replacing any non-escaped ampersand characters with the matched string
     # - For the replacement text to be multi-line, an \ needs to be specified to escape the newline character
-    local -r conf_to_replace="if (empty(\$_SERVER['HTTP_HOST'])) {\\
+    local conf_to_replace="if (empty(\$_SERVER['HTTP_HOST'])) {\\
   \$_SERVER['HTTP_HOST'] = '127.0.0.1:${http_port}';\\
-}\\
+}"
```

Comment:
I think should be 
```
}"\\
```
in order to continue the code block.

Reply:
You don't need the backslashes outside the string. The `"` ends the string. The backslashes are for escaping LF for `sed` within the string.